glue/main.py

```diff
@@ -177,10 +185,12 @@ def start_glue(gluefile=None, config=None, datafiles=None):
 
     hub = data.hub
 
+    splash.set_progress(100)
+
     session = glue.core.Session(data_collection=data, hub=hub)
     ga = GlueApplication(session=session)
     #ga.show()
-    #splash.close()
+    splash.close()
```

Comment:
As discussed elsewhere, the reason we commented this out originally was that on some platforms (the binary app, I'm pretty sure), the close() command wasn't running for some reason. As long as we confirm that that no longer happens, I'm +1 on this. The splash screen is so cool